src/screens/homepage/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { View, Text } from 'react-native'
```

Comment:
@shreya-mishra @Rohit2698 Are we going to add unit tests from the starting? If yes, we should add some tests as well.